operators/Makefile

```diff
@@ -38,7 +38,10 @@ endif
 IMG_SUFFIX ?= -$(subst _,,$(USER))
 IMG ?= $(REGISTRY)/$(REPOSITORY)/$(NAME)$(IMG_SUFFIX)
 TAG ?= $(shell git rev-parse --short --verify HEAD)
-OPERATOR_IMAGE ?= $(IMG):$(TAG)
+OPERATOR_IMAGE ?= $(IMG):$(VERSION)-$(TAG)
+ifeq ($(SNAPSHOT_RELEASE), false)
+	OPERATOR_IMAGE = $(IMG):$(TAG)
+endif
```

Comment:
I think it's nice to keep the version in the image name (`docker.elastic.co/eck/eck-operator:0.9.0-SNAPSHOT-24a02a0b`). What motivates you to remove it?

Reply:
It's only for release, because for release it looks like `docker.elastic.co/eck/eck-operator:0.8.0` where `0.8.0` tag in Github

Reply:
I guess it's to be in sync with the name used in `LATEST_RELEASED_IMG`? It looks like dangerous.
Isn't it time to remove `LATEST_RELEASED_IMG`?

Reply:
Ok, I read in the wrong way.